src/backend/marsha/core/serializers.py

```diff
@@ -141,8 +143,10 @@ def create(self, validated_data):
             validated_data["video_id"] = user.id
         return super().create(validated_data)
 
-    def get_url(self, obj):
-        """Url of the timed text track, signed with a CloudFront key if activated.
+    def get_origin_url(self, obj):
```

Comment:
origin_url was hard to understand by its name. We are calling it "source" elsewhere?

Reply:
:+1: